code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -287,10 +293,38 @@
 			L.Paralyse((preggers/10)+10) //something like 25 ticks = 20 seconds with the default settings
 	else if (iscorgi(M))
 		var/mob/living/simple_animal/corgi/C = M
+		var/obj/item/clothing/head/headwear = C.inventory_head
+		var/rng = 100
+		var/obj/item/clothing/mask/facehugger/hugger = C.facehugger
+
+		if(hugger && !hugger.sterile)
+			return
+
+		if(headwear)
+			if(istype(headwear, /obj/item/clothing/head/cardborg))
+				rng = CHANCE_TO_REMOVE_HEADWEAR
+			else if(istype(headwear, /obj/item/clothing/head/helmet/space/rig))
+				rng = CHANCE_TO_REMOVE_SPECIAL_HEADWEAR
+
+			if(prob(rng))
+				C.visible_message("<span class='danger'>\The [src] smashes against [C]'s \the [headwear], and rips it off in the process!</span>")
+				C.drop_from_inventory(headwear)
+				GoIdle(TIME_IDLE_AFTER_HEAD_DENIED)
+				return
+			else
+				C.visible_message("<span class='danger'>\The [src] bounces off of the [headwear]!</span>")
```

Comment:
\the

Reply:
So \The is redundant here, or is this a case-sensitivity thing?

Reply:
`the [headwear]` will print `the the security helmet`
make it `\the [headwear]`

Reply:
Whoops